include/cocaine/rpc/dispatch.hpp

```diff
@@ -285,6 +292,7 @@ struct slot_builder<Event, executor_state<F, R>> {
     ~slot_builder() noexcept(false) {
         typedef typename aux::select<
             R,
+            typename std::remove_reference<decltype(fn)>::type,
```

Comment:
There is no lambda: https://github.com/3Hren/cocaine-plugins/blob/develop/logging/src/logging_v2.cpp#L526